contracts/gateway/SimpleStake.sol

```diff
@@ -15,101 +15,133 @@ pragma solidity ^0.5.0;
 // limitations under the License.
 //
 // ----------------------------------------------------------------------------
-// ValueChain: SimpleStake
-// 
+//
 // http://www.simpletoken.org/
 //
 // ----------------------------------------------------------------------------
 
 import "./EIP20Interface.sol";
 import "../lib/SafeMath.sol";
-import "./ProtocolVersioned.sol";
 
 /**
- *  @title SimpleStake contract which implements ProtocolVersioned.
+ *  @title SimpleStake contract
  *
- *  @notice Stakes the value of an EIP20 token on Ethereum
- *          for a utility token on the OpenST platform.
+ *  @notice This holds staked EIP20 tokens for a gateway.
  */
-contract SimpleStake is ProtocolVersioned {
+contract SimpleStake {
     using SafeMath for uint256;
 
+
     /* Events */
 
-    event ReleasedStake(address indexed _protocol, address indexed _to, uint256 _amount);
+    /* Emitted when amount is un-staked */
+    event ReleasedStake(
+        address indexed _gateway,
+        address indexed _to,
+        uint256 _amount
+    );
+
 
     /* Storage */
 
     /** EIP20 token contract that can be staked. */
     EIP20Interface public eip20Token;
```

Comment:
Should we call this `token`?
`eip20Token`  -> `token`.
If yes then through out the file.